### PR TITLE
Branding compliance updates

### LINKED ISSUE #39: branding compliance
- every reference to an actual BIG-IP (not a BigIP object) needs to be in caps and have the appropriate trademark.
- remove 'device' after BIG-IP (it's normally used to refer to hardware, and we're referring to either hardware or VE).

### PR DESCRIPTION
**Pull request by jputrino**
@pjbreaux 

Fixes #39
Problem: Repo needs to be brought into compliance with F5 branding requirements.
Analysis: N/A
Unit/Robot tests: N/A

#### What's this change do?
- adds appropriate trademarks to every reference to F5, F5 Networks, iApp/s, iControl, BIG-IP, etc.
- I also updated the link in index.rst to the versions, releases, and support matrix.

#### Where should the reviewer start?
- view the changes below or build the docs locally (git clone my fork, cd docs, make html, view docs in browser)

#### Any background context?

### CODE REVIEW

File: unsupported/f5_plugins/apic_iapp/iapps/hitesh.tmpl [hunk 1]
```diff
@@ -238,7 +238,7 @@ proc get_var { name } {
 # Input: $name = name of variable    
 #        $checkvalue = the string that disables the option
 #        $option = TMSH name of the option
-#        $module = the BIG-IP module that enables the option
+#        $module = the BIG-IP®module that enables the option
```
Comment (pjbreaux) [inline unsupported/f5_plugins/apic_iapp/iapps/hitesh.tmpl]: Hmm is it possible to remove this? I'm not sure what will happen if we have this in an iApp template. This is technically code.

File: unsupported/f5_plugins/apic_iapp/iapps/hitesh.tmpl [hunk 2]
```diff
@@ -259,7 +259,7 @@ proc handle_opt_remove_on_redeploy { name checkvalue option module } {
 }
 
 # Check whether a specified module is provisioned and at what levels
-# Adapted from original code including the F5 iApp TCL helper library
+# Adapted from original code including the F5® iApp TCL helper library
```
Comment (pjbreaux) [inline unsupported/f5_plugins/apic_iapp/iapps/hitesh.tmpl]: Same with this one as above.

File: unsupported/f5_plugins/apic_iapp/iapps/hitesh.tmpl (hunk 1 above)
Comment (jputrino) [inline unsupported/f5_plugins/apic_iapp/iapps/hitesh.tmpl]: Yeah, that one shouldn't have been added. Good catch!

File: unsupported/f5_plugins/apic_iapp/iapps/hitesh.tmpl (hunk 2 above)
Comment (jputrino) [inline unsupported/f5_plugins/apic_iapp/iapps/hitesh.tmpl]: fixed.

**Comment by pjbreaux** [comment]
Ship it!

**Pull request merged**